spec/fixtures/static_pages/mp.php?mpn=Christine_Milne&mpc=Tasmania&house=senate&display=everyvote.html

```diff
@@ -106,14 +106,14 @@ <h2 class="social-share-heading small">
 
 <h2 id="divisions">All Votes</h2>
 <p>All votes Senator Christine Milne could have attended.</p>
-<ol class="display-house-all divisions-list list-unstyled">
+<ol class="display-house-senate divisions-list list-unstyled">
```

Comment:
I am not sure what to do about these tests. Most changes seems reasonable, like deleting votes that are from different years and adding the pagination section. However, I am not sure why this class has changed or why the `span` below lost its text.